frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -691,3 +703,103 @@ export function aggregateCollectionsFromDatasets(
 
   return collections;
 }
+
+interface MarkerGeneRequestBody {
+  celltype: string;
+  n_markers: number;
+  organism: string;
+  test: string;
+  tissue: string;
+}
+
+interface HardcodedMarkerGeneRequest extends MarkerGeneRequestBody {
+  n_markers: 10;
+  test: "ttest";
+}
+
+export function generateMarkerGeneBody(
+  cellTypeID: string,
+  tissueID: string,
+  organismID: string
+): HardcodedMarkerGeneRequest | null {
```

Comment:
hmm I might be wrong, but this doesn't seem to return `null`?

```suggestion
): HardcodedMarkerGeneRequest {
```